docs/_docs/upgrading.md

```diff
@@ -8,3 +8,5 @@ Upgrading from an older version of Jekyll? Upgrading to a new major version of J
 
 - [From 0.x to 1.x and 2.x](/docs/upgrading/0-to-2/)
 - [From 2.x to 3.x](/docs/upgrading/2-to-3/)
+
+If you are making a minor update (for example from 3.3.1  to 3.3.2 ) run 'gem update jekyll' when in your site directory. 
```

Comment:
## Keeping Jekyll up to date

If you followed our setup recommendations and installed Bundler, run `bundle update jekyll` or simply `bundle update` and all your gems will update to the latest versions.
If you don't have Bundler installed, run `gem update jekyll`